amy/templates/includes/trainingrequest_details.html

```diff
@@ -30,6 +30,8 @@
     <tr><th>Continue with registration code marked as invalid:</th>
         <td>{{ object.member_code_override|yesno }}</td></tr>
 {% endif %}
+    <tr><th>Eventbrite URL:</th>
+        <td>{{ object.eventbrite_url|default:"&mdash;" }}</td></tr>
```

Comment:
You can make it clickable, but then I'd for sure use domain whitelisting to make sure someone doesn't use spoof the original domain (e.g. by using homoglyphs).

Reply:
Yeah, I spoke about this with Rob last week and we thought that doing this safely would be more effort than it's worth. I don't anticipate these links being used directly very often, their main purpose is to be searchable.